src/anbox/build/version.h.in

```diff
@@ -17,11 +17,22 @@
  *
  */
 
-#include "anbox/version.h"
+#ifndef ANBOX_VERSION_H_
+#define ANBOX_VERSION_H_
 
-void anbox::version(std::uint32_t& major, std::uint32_t& minor,
-                    std::uint32_t& patch) {
-  major = anbox::build::version_major;
-  minor = anbox::build::version_minor;
-  patch = anbox::build::version_patch;
-}
+#include <cstdint>
+#include <string>
+
+namespace anbox {
+namespace build {
+/// @brief version_major marks the major version
+static constexpr const std::uint32_t version_major{@VERSION@};
+/// @brief version_suffix is an additional suffix which can be amended to
+/// the major version number to indicate a dev build for example.
+static const std::string version_suffix{"@VERSION_SUFFIX@"};
```

Comment:
I would propose to make this static constexpr const char* version_suffix, thereby avoiding static initialization fiascos.